app/assets/images/pushpin.svg

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
You can minify and clean this SVG up automatically with Jake Archibald's [SVGOMG](https://jakearchibald.github.io/svgomg/) tool. I ran it through the tool and brought the SVG from 977bytes to 706bytes. Since I know you're all about the perf, every little bit helps. 😉

Reply:
Fixed 💪